lib/backend/sqlite.cc

```diff
@@ -194,8 +209,19 @@ static int sqlite_fini(rpmdb rdb)
 	    rdb->db_opens--;
 	} else {
 	    if (sqlite3_db_readonly(sdb, NULL) == 0) {
+
 		sqlexec(sdb, "PRAGMA optimize");
 		sqlexec(sdb, "PRAGMA wal_checkpoint = TRUNCATE");
+
+		int max_size = rpmExpandNumeric("%{?_sqlite_vacuum_kb}");
+		if (max_size <= 0)
+		    max_size = 20*1024;
+		int64_t free_space = sqlite_free_space(sdb) >> 10;
```

Comment:
int64_t here doesn't help when it's long in sqlite_free_space(). Which should return the same kind of data as the macro really so this calculation belongs to that function, more like. Doing *1024 but then >> 10 looks super confusing, lets just stick to basic maths for doing math, because to me a bitshift suggests there's something more special going on.